lib/data_update_scripts/20200708163323_backfill_broadcastable_type_for_broadcasts.rb

```diff
@@ -0,0 +1,9 @@
+module DataUpdateScripts
+  class BackfillBroadcastableTypeForBroadcasts
+    def run
+      Broadcast.where(broadcastable_type: nil).each do |cast|
```

Comment:
this should probably use `find_each` but as I don't think we have so much data it shouldn't be an issue :D